astronomer/providers/apache/livy/hooks/livy.py

```diff
@@ -0,0 +1,410 @@
+"""This module contains the Apache Livy hook async."""
+import asyncio
+import re
+from typing import Any, Dict, List, Optional, Sequence, Union
+
+import aiohttp
+from aiohttp import ClientResponseError
+from airflow.providers.apache.livy.hooks.livy import BatchState
+from airflow.utils.log.logging_mixin import LoggingMixin
+from asgiref.sync import sync_to_async
+
+from astronomer.providers.http.hooks.http import HttpHookAsync
+
+
+class LivyHookAsync(HttpHookAsync, LoggingMixin):
```

Comment:
why you are not inheriting this from `LivyHook` ?https://github.com/apache/airflow/blob/602abe8394fafe7de54df7e73af56de848cdf617/airflow/providers/apache/livy/hooks/livy.py#L46

Reply:
I don't need the functionality of `LivyHook`. It would suit more for `HttpHookAsync` as that's the method I use to may REST calls